Makefile

```diff
@@ -21,7 +21,7 @@ all: init frontend install
 
 .PHONY: all-devel
 # Get dependencies and install Streamlit into Python environment -- but do not build the frontend.
-all-devel: init develop
+all-devel: init develop pre-commit-install
```

Comment:
I don't think we need to call `pre-commit-install` since it is already part of `init`, or? On the other hand, maybe we even remove it from `init` and only have it in our `devel` (`all-devel` and `mini-devel`) calls since this is clearly a development targeted dependency.

Reply:
+1 to calling `pre-commit-install` in the two devel targets and not `init`

Reply:
@LukasMasuch @vdonato good point 👍 I've just changed it 👍